critwall/src/main/java/org/opencadc/critwall/FileSync.java

```diff
@@ -67,49 +67,48 @@
 
 package org.opencadc.critwall;
 
-import ca.nrc.cadc.reg.Capabilities;
-import ca.nrc.cadc.reg.Capability;
-import ca.nrc.cadc.reg.Standards;
-import ca.nrc.cadc.reg.client.RegistryClient;
+import ca.nrc.cadc.db.ConnectionConfig;
+import ca.nrc.cadc.db.DBUtil;
 
-import java.io.IOException;
 import java.net.URI;
 import java.util.Iterator;
 import java.util.Map;
 
+import javax.naming.NamingException;
 import org.apache.log4j.Logger;
+import org.opencadc.inventory.Artifact;
 import org.opencadc.inventory.InventoryUtil;
-import org.opencadc.inventory.SiteLocation;
 import org.opencadc.inventory.db.ArtifactDAO;
-import org.opencadc.inventory.storage.StorageAdapter;
+import org.opencadc.inventory.storage.fs.OpaqueFileSystemStorageAdapter;
+
 
-/**
- *
- * @author pdowler
- */
 public class FileSync {
     private static final Logger log = Logger.getLogger(FileSync.class);
 
     private static final int MAX_THREADS = 16;
     
     private final ArtifactDAO artifactDAO;
-    private StorageAdapter localStorage;
-    private final URI resourceID;
-    private final Capability locator;
+    private final ArtifactDAO jobArtifactDAO;
+    private final URI locatorResourceID;
```

Comment:
I noticed this rename from resourceID to locatorResourceID here and in FSJ... just to clarify? There is only one remote service (resourceID) that critwall looks up so I'm not sure it is warranted. It also means the name used in  FileSync ctor, and check (line~113) is different from the internal variable now. Admittedly, the name in the config is `locatorService` which is neither of these so a rename could be justified if the name *everywhere* was the same as the config item.

Reply:
switched names to locatorService.